clang/lib/Analysis/ExprMutationAnalyzer.cpp

```diff
@@ -100,6 +100,20 @@ AST_MATCHER(CXXTypeidExpr, isPotentiallyEvaluated) {
   return Node.isPotentiallyEvaluated();
 }
 
+AST_MATCHER(CXXMemberCallExpr, isConstCallee) {
+  const auto *CalleeDecl = Node.getCalleeDecl();
```

Comment:
Please don't use `auto` unless type is explicitly stated in same statement or iterator.